pkg/controller/clusterconfig/clusterconfig_controller.go

```diff
@@ -108,6 +109,18 @@ func (r *ReconcileClusterConfig) Reconcile(ctx context.Context, request reconcil
 	network.MergeClusterConfig(&operConfig.Spec, clusterConfig.Spec)
 
 	if _, ok := clusterConfig.Annotations[names.NetworkTypeMigrationAnnotation]; ok {
```

Comment:
interesting that we use an annotation for the clusterconfig controller side of things to signal a live migration? but the name of the annotation is generic that I don't know if its live or something else.. but unrelated to this PR and this comment is out of context and self homework for myself.

Reply:
> interesting that we use an annotation for the clusterconfig controller side of things to signal a live migration?

Yes, it gates the migration.